torch/csrc/generic/methods/TensorSerialization.cwrap

```diff
@@ -70,6 +70,11 @@ PyObject * THPTensor_(toNumpy)(THPTensor *self, PyObject *args) {
   std::unique_ptr<npy_intp[]> sizes;
   std::unique_ptr<npy_intp[]> strides;
 
+  if (!PyArg_ParseTuple(args, "")) {
```

Comment:
A faster way to do it would be `THPUtils_assert(PyTuple_SIZE(args) == 0, "numpy() ...");`